[Bug] Incorrect comment line with DICOM JSON generation instruction

### Describe the Bug

It seems that that the instruction in https://github.com/OHIF/Viewers/blob/master/.scripts/dicom-json-generator.js which says:

```
Usage: node dicomStudyToJSONLaunch.js <studyFolder> <urlPrefix> <outputJSONPath> <optional scheme>
```

is wrong. The script is now named `dicom-json-generator.js`

Same applies to following lines:

```
if (args.length < 3 || args.length > 4) {
  console.error(
    'Usage: node dicomStudyToJSONLaunch.js <studyFolder> <urlPrefix> <outputJSONPath> [scheme]'
  );
  process.exit(1);
}
```

### Steps to Reproduce

1. Clone repo
2. `yarn config set workspaces-experimental true`
3. `yarn install`
4. Run `node dicomStudyToJSONLaunch.js <studyFolder> <urlPrefix> <outputJSONPath> <optional scheme>`

### The current behavior

```
grzanka@grzankax1:~/workspace/Viewers/.scripts$ node dicomStudyToJSONLaunch.js XXX/DICOM/ https://XXXX/MR_2019/  MR_2019.json
node:internal/modules/cjs/loader:1146
  throw err;
  ^

Error: Cannot find module '/home/grzanka/workspace/Viewers/.scripts/dicomStudyToJSONLaunch.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
    at Module._load (node:internal/modules/cjs/loader:984:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.12.0
```

### The expected behavior

```
grzanka@grzankax1:~/workspace/Viewers/.scripts$ node dicom-json-generator.js XXXX/IMAGES/ XXXX/MR_2019/  MR_2019.json
Processing...
Successfully loaded data
JSON saved
```

### OS

Linux Ubuntu 24.04

### Node version

20.12.0

### Browser

not applicable

--- a/.scripts/dicom-json-generator.js
+++ b/.scripts/dicom-json-generator.js
@@ -4,7 +4,7 @@
  * The JSON file can be used to load the study into the OHIF Viewer. You can get more detail
  * in the DICOM JSON Data source on docs.ohif.org
  *
- * Usage: node dicomStudyToJSONLaunch.js <studyFolder> <urlPrefix> <outputJSONPath> <optional scheme>
+ * Usage: node dicom-json-generator.js <studyFolder> <urlPrefix> <outputJSONPath> <optional scheme>
  *
  * params:
  * - studyFolder: path to the study folder which contains the DICOM files
@@ -30,7 +30,7 @@ const [studyDirectory, urlPrefix, outputPath, scheme = 'dicomweb'] = args;
 
 if (args.length < 3 || args.length > 4) {
   console.error(
-    'Usage: node dicomStudyToJSONLaunch.js <studyFolder> <urlPrefix> <outputJSONPath> [scheme]'
+    'Usage: node dicom-json-generator.js <studyFolder> <urlPrefix> <outputJSONPath> [scheme]'
   );
   process.exit(1);
 }

